libcst/codegen/transforms.py

```diff
@@ -0,0 +1,18 @@
+import libcst as cst
+
+
+class SimplifyUnionsTransformer(cst.CSTTransformer):
+    def leave_Subscript(
+        self, original_node: cst.Subscript, updated_node: cst.Subscript
+    ) -> cst.BaseExpression:
+        if updated_node.value.deep_equals(cst.Name("Union")):
```

Comment:
Let's use matcher?
```suggestion
        if m.matches(updated_node, m. Subscript(value=m.Name("Union")):
```

Reply:
We should not use matchers in codegen because it couples more of LibCST into codegen of LibCST itself. I intentionally use the least amount of it as I can get away with.